spec/bcx/comment_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'spec_helper'
+
+describe Bcx::Resources::Comment, :vcr do
+  let(:client) { Bcx::Client::HTTP.new(login: 'bcx-test-user', password: 'secret') }
```

Comment:
Line is too long. [84/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.